dipy/denoise/noise_estimate.py

```diff
@@ -149,10 +146,11 @@ def _piesno_3D(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5,
     -----------
     data : ndarray
         The magnitude signals to analyse. The last dimension must contain the
-        same realisation of the volume, such as dMRI or fMRI data.
+        same realisation of the volume.
```

Comment:
Maybe use the same wording as above? ("e.g. volumes collected during dMRI or fMRI acquisitions.")